.github/ISSUE_TEMPLATE/release-checklist.md

```diff
@@ -34,12 +34,14 @@ GitHub release:
  - [ ] Wait until the Bodhi update shows "Signed :heavy_check_mark:" in the Metadata box.
  - [ ] [File a releng ticket](https://pagure.io/releng/new_issue) based on [prior signing tickets](https://pagure.io/releng/issue/10158).
    - [ ] Update the script and test it locally by dropping the `sigul` lines.
+   - [ ] If a new Fedora release has gone stable, update the signing key in the script to use the new Fedora signing key [found here](https://getfedora.org/security).
```

Comment:
This should happen before "test it locally", or we could just indent by one level.